code-studio/src/js/components/progress/course_progress.jsx

```diff
@@ -0,0 +1,42 @@
+/* global React, dashboard */
+
+window.dashboard = window.dashboard || {};
+
+/**
+ * Stage progress component used in level header and course overview.
+ */
+window.dashboard.CourseProgress = (function (React) {
```

Comment:
We don't need to stick these on window.dashboard anymore, right? We can just export the component, then require it where we need it.